lib/rubocop/cop/style/empty_else.rb

```diff
@@ -4,11 +4,19 @@ module RuboCop
   module Cop
     module Style
       # Checks for empty else-clauses, possibly including comments and/or an
-      # explicit `nil`.
+      # explicit `nil` depending on the EnforcedStyle.
+      #
+      # SupportedStyles:
+      # implicit - warn on empty else and missing else
```

Comment:
The names of the styles are a bit confusing - someone might expect that `implicit` will actually enforce `implicit` else clauses.